relays/bin-substrate/src/rialto_millau/rococo_headers_to_westend.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright 2019-2021 Parity Technologies (UK) Ltd.
+// This file is part of Parity Bridges Common.
+
+// Parity Bridges Common is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity Bridges Common is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity Bridges Common.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Rococo-to-Westend headers sync entrypoint.
+
+use crate::finality_pipeline::{SubstrateFinalitySyncPipeline, SubstrateFinalityToSubstrate};
+
+use bp_header_chain::justification::GrandpaJustification;
+use codec::Encode;
+use relay_rococo_client::{Rococo, SyncHeader as RococoSyncHeader};
+use relay_substrate_client::{Chain, TransactionSignScheme};
+use relay_utils::metrics::{FloatJsonValueMetric, MetricsParams};
+use relay_westend_client::{SigningParams as WestendSigningParams, Westend};
+use sp_core::{Bytes, Pair};
+
+/// Rococo-to-Westend finality sync pipeline.
+pub(crate) type RococoFinalityToWestend = SubstrateFinalityToSubstrate<Rococo, Westend, WestendSigningParams>;
+
+impl SubstrateFinalitySyncPipeline for RococoFinalityToWestend {
+	const BEST_FINALIZED_SOURCE_HEADER_ID_AT_TARGET: &'static str = bp_rococo::BEST_FINALIZED_ROCOCO_HEADER_METHOD;
+
+	type TargetChain = Westend;
+
+	fn customize_metrics(params: MetricsParams) -> anyhow::Result<MetricsParams> {
+		Ok(
```

Comment:
I'd either remove them from Westend -> Millau headers relay (would need a change in Grafana/Prometheus config), or from here. Why would we need two samples of "just-for-test" metrics?

Reply:
I expect that it's going to be different people looking at these metrics (and they will be visibile in dfferent places).
1. We have bridge-team-internal Grafana and our team looking at Westend<>Millau stuff.
2. We have parity-internal Grafana and our devops team looking at Rococo<>Westend.

Hence I'd keep both. I extracted a function though, so the code is not triplicated.